scholia/app/templates/base.html

```diff
@@ -87,6 +87,11 @@
 
 <script type="text/javascript">
     {% if q %}
+    if ( $( "#bioschemas" ).length ) {
+      var bioschemasurl = "{{ url_for('app.index') }}{{ q }}/bioschemas";
```

Comment:
should be able to do the following, which knows how to automatically construct the URL based on the name of the function registered to the app blueprint (which is strangely named `main` and not `app`...)

```suggestion
      var bioschemasurl = "{{ url_for('app.bioschemas', q=q) }}";
```

It works because even though the endpoint name is constructed through semi-dynamic string concatenation, it still knows tha variable for the function is called `q`.

Reply:
feel free to make PRs against this PR again :)

Reply:
You should be able to directly accept/commit the suggestions from the github PR interface, which is effectively the same

Reply:
I guess that lacks the benefit of me actually having to test it first 🙃

Reply:
oh, lol, I did not know that! cool! I now see the button, but I guess I visually ignored it (pretty much like how my brain automatically ignores a lot of spam on webpages).

Reply:
`app.bioschemas_for` fixed (patch pending)